### PR TITLE
Automate aur package publish after each release

### PR DESCRIPTION
**Pull request by olblak**
Signed-off-by: Olblak <me@olblak.com>

Fix https://github.com/updatecli/website/pull/651

This pullrequest is not tested, I'll probably use the 0.45.0 alone

## Test

/

### CODE REVIEW

**Review by dduportal — DISMISSED** [review]

**Comment by olblak** [comment]
I am waiting for some news from @jneidel before merging this pullrequest

https://github.com/updatecli/website/pull/651#issuecomment-1429952205

File: .goreleaser.yml
Comment (jneidel) [inline .goreleaser.yml:20]: `A declarative dependency management command line tool.` is sufficient :smile:

File: .goreleaser.yml
Comment (jneidel) [inline .goreleaser.yml:23]: `Apache`

File: .goreleaser.yml
Comment (jneidel) [inline .goreleaser.yml:34]: Redundant as you're using `install -D`

File: .goreleaser.yml
Comment (jneidel) [inline .goreleaser.yml:30]: Use `${pkgname}` instead of the literal `updatecli`

File: .goreleaser.yml
Comment (jneidel) [inline .goreleaser.yml:28]: Use `${pkgname}` instead of the literal `updatecli`

**Review by jneidel — CHANGES_REQUESTED** [review]

File: .goreleaser.yml
Comment (jneidel) [inline .goreleaser.yml:22]: You can also add me here :smile: : `Jonathan Neidel <aur at jneidel dot com>`

**Comment by jneidel** [comment]
I've added you (`olblak`) as a co-maintainer. So after the changes feel free to release 0.44.3.

Let me know here once you've done so and I'll test the package and verify everything is working as intended :+1:

File: .goreleaser.yml
Comment (olblak) [reply .goreleaser.yml:20]: Thanks for the feedback

Comment (olblak) [reply .goreleaser.yml:20]:
```suggestion
    description: A declarative dependency management tool.
```

File: .goreleaser.yml
Comment (olblak) [reply .goreleaser.yml:22]: Awesome, I was looking for that email \o/

Comment (olblak) [reply .goreleaser.yml:22]:
```suggestion
      - 'Olivier Vernin <olivier at vernin dot me>'
      - 'Jonathan Neidel <aur at jneidel dot com>'
```

File: .goreleaser.yml
Comment (olblak) [reply .goreleaser.yml:23]:
```suggestion
    license: 'Apache'
```

File: .goreleaser.yml
Comment (olblak) [reply .goreleaser.yml:28]:
```suggestion
      install -Dm755 "./updatecli" "${pkgdir}/usr/bin/${pkgname}"
```

File: .goreleaser.yml
Comment (olblak) [reply .goreleaser.yml:30]:
```suggestion
      install -Dm644 "./LICENSE" "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
```

**Comment by olblak** [comment]
@jneidel I had a quick look at the package that you published 

```
pkgname=updatecli-bin
pkgver="0.44.0"
pkgrel=1
pkgdesc="A Declarative Dependency Management tool"
arch=(x86_64 aarch64)
url="https://github.com/updatecli/updatecli"
license=("Apache")
depends=()
makedepends=("git")
optdepends=()
provides=(updatecli)
source_x86_64=("https://github.com/updatecli/updatecli/releases/download/v${pkgver}/updatecli_Linux_x86_64.tar.gz")
source_aarch64=("https://github.com/updatecli/updatecli/releases/download/v${pkgver}/updatecli_Linux_arm64.tar.gz")
sha256sums_x86_64=("55502f735a1ca7d752198bca65264051e27f2dcf386149167efd3724f4878e85")
sha256sums_aarch64=("1fbb15c5cab4f0c31ad3cc9ea0274c5cfe1bdf2c84be07113743439093615787")

package() {
  cd "$srcdir"

  install -Dm755 updatecli "$pkgdir/usr/bin/updatecli"
  install -Dm644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
  install -Dm644 manpages/updatecli.1.gz "$pkgdir/usr/share/man/man1/updatecli.1.gz"
  install -Dm644 completions/updatecli.bash "$pkgdir/usr/share/bash-completion/completions/updatecli"
  install -Dm644 completions/updatecli.fish "$pkgdir/usr/share/fish/vendor_completions.d/updatecli.fish"
  install -Dm644 completions/updatecli.zsh "$pkgdir/usr/share/zsh/site-functions/_updatecli"
}
```

My understanding is pkgname is set to `updatecli-bin` but we want to use `updatecli` so we shouldn't use pkgname in the script. Am I right?

**Comment by olblak** [comment]
I am waiting for the tests to pass then I merge and trigger a release 0.44.3

**Comment by jneidel** [comment]
Can confirm that 0.44.4 works on install and that PKGBUILD looks good :smile:

**Pull request merged**
